From e3c91039d8fc04d6d914c34eb77b0e42e326a2d2 Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.2-e3c91039d8f-20220110T220553Z-avarab@gmail.com>
In-Reply-To: <cover-0.2-00000000000-20220110T220553Z-avarab@gmail.com>
References: <CANYiYbEYgSCx230S29zVhMKb8J8WQ1ScxVHn6fMvdhPOdjpBCg@mail.gmail.com>
	<cover-0.2-00000000000-20220110T220553Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 10 Jan 2022 17:38:05 +0100
Subject: [PATCH 2/2] cat-file: s/_/-/ in typo'd usage_msg_optf() message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a typo in my recent 03dc51fe849 (cat-file: fix remaining usage
bugs, 2021-10-09).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/cat-file.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builtin/cat-file.c b/builtin/cat-file.c
index e36492235ba..7b3f42950ec 100644
--- a/builtin/cat-file.c
+++ b/builtin/cat-file.c
@@ -727,7 +727,7 @@ int cmd_cat_file(int argc, const char **argv, const char *prefix)
 		;
 	else if (batch.follow_symlinks)
 		usage_msg_optf(_("'%s' requires a batch mode"), usage, options,
-			       "--follow_symlinks");
+			       "--follow-symlinks");
 	else if (batch.buffer_output >= 0)
 		usage_msg_optf(_("'%s' requires a batch mode"), usage, options,
 			       "--buffer");
-- 
2.34.1.1373.g062f5534af2

